spec/models/incidental_type_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe IncidentalType, type: :model do
+  context 'will properly do validations' do
+    it 'builds an IncidentalType' do
+      expect(build :incidental_type).to be_valid
+    end
+    it 'fails to build when it is missing data' do
+      expect(build :incidental_type, name: nil).not_to be_valid
+      expect(build :incidental_type, description: nil).not_to be_valid
+      expect(build :incidental_type, base: nil).not_to be_valid
+      expect(build :incidental_type, modifier_amount: nil).not_to be_valid
+      expect(build :incidental_type, modifier_description: nil).not_to be_valid
+    end
+    it 'wont create two types with the same name' do
+      same = create :incidental_type, name: 'same'
```

Comment:
It's better to only specify model attributes when there's something significant about the value(s). Since your test should work regardless of what the incidental type's name is, better not to write a specific one.

In particular, I could write the following code:

```ruby
class IncidentalType < ActiveRecord::Base
  validate :name_is_not_same

  # rest of model code

  private

  def name_is_not_same
    if name == 'same'
      errors.add :name, 'cannot be the word same'
    end
  end
end
```

which would make your test pass for the wrong reason.